templates/docs/examples/patterns/side-navigation/_toggle_script.js

```diff
@@ -1,10 +1,70 @@
+// This is not a production ready code, just serves as an example
+// of how the focus should be controlled within the sidenav drawer.
+var sidenavContainer = null;
```

Comment:
Just a small change, but I would rename it to `expandedSidenavContainer` to make it a bit clearer.